### PR TITLE
[#155512387] Enable admin to assign an asset to an existing user[WIP]

### PR DESCRIPTION
**Pull request by jmnyarega**
### What does this PR do?
- Create an association between `user` and `item` models

### How can I manually test this?
- `python manage.py test`
- Login to your admin account
- Under the `api` section in the admin dashboard, click on `Items` and then add a new item
- Save the `Item` details

### CODE REVIEW

File: api/models.py [hunk 1]
```diff
@@ -72,6 +72,9 @@ class Item(models.Model):
     serial_number = models.CharField(max_length=50, blank=True)
     created_at = models.DateTimeField(auto_now_add=True, editable=False)
     last_modified = models.DateTimeField(auto_now=True, editable=False)
+    user_id = models.ForeignKey('User',
```
Comment (egichuri) [inline api/models.py]: we could call this `assigned_to`

**Comment by egichuri** [comment]
@jmnyarega what's remaining for this?

File: api/models.py (hunk 1 above)
Comment (jmnyarega) [reply api/models.py]: Changed

**Comment by jmnyarega** [comment]
@egichuri, It's done

**Comment by HawiCaesar** [comment]
LGTM

**Comment by jmnyarega** [comment]
@alexkiura @egichuri Kindly Review

**Pull request merged**